app/src/main/java/com/example/mycalculator/mvp/CalculatorView.java

```diff
@@ -0,0 +1,19 @@
+package com.example.mycalculator.mvp;
```

Comment:
yo can create a package called view, and put this class inside

Reply:
Done. Thx!